packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -284,6 +284,7 @@ export default class CFDStore extends BaseStore {
             }
         } else if (platform === CFD_PLATFORMS.CTRADER) {
             startPerformanceEventTimer('create_ctrader_account_time');
+            this.root_store.client.setTradersHubLoading(false);
```

Comment:
It should be true at the start of function and set to false at the end of it. All loaders are set this way.

Reply:
I renamed the function name. Since it was creating confusion.